# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/pkg_patch/pkg_udev.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- udev-083/extras/scsi_id/scsi.h.orig	2006-01-27 01:41:07.000000000 +0100
+++ udev-083/extras/scsi_id/scsi.h	2006-02-01 16:44:28.000000000 +0100
@@ -81,9 +81,9 @@
 #define	SCSI_ID_ASCII	2
 
 struct scsi_id_search_values {
-	u_char	id_type;
-	u_char	naa_type;
-	u_char	code_set;
+	unsigned char	id_type;
+	unsigned char	naa_type;
+	unsigned char	code_set;
 };
 
 /*
